registration/src/test/java/org/spine3/samples/lobby/registration/procman/Given.java

```diff
@@ -0,0 +1,270 @@
+            return RegistrationProcess.getDefaultInstance();
+        }
+
+        // Is overridden to make accessible in tests.
+        @Override
+        public void incrementState(RegistrationProcess newState) {
+            super.incrementState(newState);
+        }
+
+        @Override
+        @SuppressWarnings("RefusedBequest") // is overridden to do not throw an exception
+        protected CommandBus getCommandBus() {
+            return boundedContext.getCommandBus();
+        }
+
+        public List<Message> getCommandsSent() {
+            return ImmutableList.copyOf(commandsSent);
+        }
+
+        private class MockCommandSender extends CommandSender {
+
+            @Override
+            protected void send(Message commandMessage) {
+                commandsSent.add(commandMessage);
+                super.send(commandMessage);
+            }
+        }
+    }
+
+    /*package*/ static class Event {
+
+        /*package*/ static final EventContext CONTEXT = EventContext.getDefaultInstance();
+
+        private static final List<SeatQuantity> SEATS = ImmutableList.of(newSeatQuantity(5), newSeatQuantity(10));
+
+        private Event() {}
+
+        /*package*/ static OrderPlaced orderPlaced() {
+            final Timestamp afterNow = minutesAhead(15);
+            return orderPlaced(afterNow);
+        }
+
+        /*package*/ static OrderPlaced orderPlaced(Timestamp reservationExpiration) {
+            final OrderPlaced.Builder builder = OrderPlaced.newBuilder()
+                    .setOrderId(ORDER_ID)
+                    .setConferenceId(CONFERENCE_ID)
+                    .addAllSeat(SEATS)
+                    .setReservationAutoExpiration(reservationExpiration);
+            return builder.build();
+        }
+
+        /*package*/ static OrderUpdated orderUpdated() {
+            final OrderUpdated.Builder builder = OrderUpdated.newBuilder()
+                    .setOrderId(ORDER_ID)
+                    .addAllSeat(SEATS);
+            return builder.build();
+        }
+
+        /*package*/ static SeatsReserved seatsReserved() {
+            final SeatsReserved.Builder builder = SeatsReserved.newBuilder()
+                    .setReservationId(RESERVATION_ID)
+                    .setConferenceId(CONFERENCE_ID)
+                    .addAllReservedSeatUpdated(SEATS);
+            return builder.build();
+        }
+
+        /*package*/ static PaymentCompleted paymentCompleted() {
+            final PaymentCompleted.Builder builder = PaymentCompleted.newBuilder()
+                    .setOrderId(ORDER_ID);
+            return builder.build();
+        }
+
+        /*package*/ static OrderConfirmed orderConfirmed() {
+            final OrderConfirmed.Builder builder = OrderConfirmed.newBuilder()
+                    .setOrderId(ORDER_ID)
+                    .addAllSeat(SEATS);
+            return builder.build();
+        }
+    }
+
+    /*package*/ static class Command {
```

Comment:
Is this class named such to be able to write `Given.Command.expireRegistrationProcess()`? If so, please document this.